styx-scheduler-service/src/main/java/com/spotify/styx/BackfillTriggerManager.java

```diff
@@ -123,15 +124,13 @@ private void triggerBackfill(Backfill backfill, Map<String, Long> backfillStates
         // Wait for the trigger execution to complete before proceeding to the next partition
         processed.get();
       } catch (AlreadyInitializedException e) {
-        LOG.warn("tried to trigger backfill for already active state [{}]: {}",
-                 partition, backfill);
-      } catch (InterruptedException e) {
-        Thread.currentThread().interrupt();
-        throw new RuntimeException(e);
+        LOG.warn("tried to trigger backfill for already active state [{}]: {}", partition, backfill);
       } catch (ExecutionException e) {
-        LOG.error("failed to trigger backfill for state [{}]: {}",
-                  partition, backfill);
-        throw new RuntimeException(e);
+        LOG.warn("failed to trigger backfill for state [{}]: {}", partition, backfill, e);
+        return;
+      } catch (Throwable e) {
+        LOG.warn("backfill triggering threw exception for state [{}]: {}", partition, backfill, e);
```

Comment:
Isn't it confusing to have distinct warnings for `ExecutionException` and `Throwable`? The rephrase of Throwable into "threw exception" isn't helping either...